astropy/utils/data_info.py

```diff
@@ -685,6 +685,17 @@ def getattrs(col):
 
         return out
 
+    def get_sortable_arrays(self):
+        """
+        Return a list of arrays which can be lexically sorted to represent
+        the order of the parent column.
+
+        The base method raises NotImplementedError and must be overridden.
+
+        :return: list of Column
```

Comment:
Here, definitely delete it!

Reply:
Left this (but with the right docstring format).  I think the base function signature should still be correct, even if it is `NotImplemented`.  That makes it easier for subclasses to get the signature right by copying.